lib/ansible/module_utils/aci.py

```diff
@@ -58,9 +58,10 @@
     timeout=dict(type='int', default=30),
     use_proxy=dict(type='bool', default=True),
     use_ssl=dict(type='bool', default=True),
-    validate_certs=dict(type='bool', default=True),
```

Comment:
So, PEP8 prefers a trailing comma when using newline list items. Mostly because it keeps the diff's concise and mergable.

Reply:
oops, that must not be on the dev repo. I updated this request.